src/Bicep.Core/CoreResources.resx

```diff
@@ -183,7 +183,7 @@
     <value>Remove unnecessary string interpolation.</value>
   </data>
   <data name="UnusedVariableRuleDescription" xml:space="preserve">
-    <value>No unused variables.</value>
+    <value>All variables must be used.</value>

```

Comment:
My only hesitation is that the linter rule id is no-unused-variables.  Is it better to keep the rule name consistent with the id?

Reply:
I think it's more clear and general good practice to use the positive version of statement. Instead of No cake was uneaten, I feel it's better to say "All cake was eaten". So I say keep it this way even though the rule id is no-unused-variables.

That is my opinion.